frontend/src/pages/MessagingQueues/MessagingQueuesUtils.ts

```diff
@@ -223,3 +224,39 @@ export const MessagingQueuesViewType = {
 		value: 'consumerLatency',
 	},
 };
+
+interface OnboardingStatusAttributeData {
+	overallStatus: string;
+	allAvailableAttributes: string[];
+	attributeDataWithError: { attributeName: string; errorMsg: string }[];
+}
+
+export const getAttributeDataFromOnboardingStatus = (
+	onboardingStatus?: OnboardingStatusResponse | null,
+): OnboardingStatusAttributeData => {
+	const allAvailableAttributes: string[] = [];
+	const attributeDataWithError: {
+		attributeName: string;
+		errorMsg: string;
+	}[] = [];
+
+	if (onboardingStatus?.data && !isEmpty(onboardingStatus?.data)) {
+		onboardingStatus.data.forEach((status) => {
+			if (status.attribute) {
+				allAvailableAttributes.push(status.attribute);
+				if (status.status === '0') {
```

Comment:
status === '0' ?

Reply:
+1

Reply:
from the API contract we have two kinds of status - `0` and `1`

Reply:
is this semantically correct